portfolio/src/main/java/com/google/sps/data/Location.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** Represents a location at a specific lat lng point. */
```

Comment:
What do you think about using the full names (latitude, longitude or x, y alternatively) both in description and the actual params? Generally we wish for names to be as descriptive as possible, and in this case saving up a few letters isn't that much of a stretch and much clearer.

Reply:
I think those are well known terms so it is descriptive and clear, like writing num instead of number.. it's not the case?  Also because all the Maps built-in fields, such as: maps.LatLng() and center: {lat: , lng: } are using it like that, I thought it's more consistent to use it too.